subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/TransformFileOperation.java

```diff
@@ -58,7 +56,7 @@ public void run(@Nullable BuildOperationContext context) {
         String displayName = "Transform " + file.getName() + " with " + transform.getDisplayName();
         return BuildOperationDescriptor.displayName(displayName)
             .progressDisplayName(displayName)
-            .operationType(category);
+            .operationType(BuildOperationCategory.UNCATEGORIZED);
```

Comment:
You don't need to call this, the default is uncategorized. But won't this make the events completely disappear from the console, too?

Reply:
It won't, all that matters is having a progressDisplayName.

This kind of confusion is why I dislike these "categories". They are only used for task grouping and updating the summary line (e.g. going from "Configuring" to "Executing")

Reply:
Exactly. The top-level things need to have category `TASK`/`TRANSFORM` to show up, everything else is shown as progress beneath if `progressDisplayName` is set.